content/blog/eng/a-redux-pattern.md

```diff
@@ -0,0 +1,203 @@
+---
+authors: ["pierroberto-lucisano"]
+comments: true
+date: "2022-04-21"
+draft: true
+share: true
+categories: [Design patterns, Javascript]
+title: "A Redux pattern"
```

Comment:
I did not know what was Redux before reading your article, so I was thinking about a pattern whose name was Redux.
Instead, you are talking about a design pattern for redux.
So what about renaming your article to something as "A pattern for using Redux"?